scripts/enableAutoComplete.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const { spawnSync } = require('child_process');
+const os = require('os');
+const fs = require('fs');
+const path = require('path');
+
+const shellConfigs = {
+  zsh: {
+    rcFile: path.join(os.homedir(), '.zshrc'),
+    detectFile: path.join(os.homedir(), '.zshrc'),
+    postMessage: 'Run: source ~/.zshrc',
+    action: (output, rcFile) => fs.appendFileSync(rcFile, `\n# AsyncAPI CLI Autocomplete\n${output}\n`),
+  },
+  bash: {
+    rcFile: path.join(os.homedir(), '.bashrc'),
+    detectFile: path.join(os.homedir(), '.bashrc'),
+    postMessage: 'Run: source ~/.bashrc',
+    action: (output, rcFile) => fs.appendFileSync(rcFile, `\n# AsyncAPI CLI Autocomplete\n${output}\n`),
+  },
+  fish: {
+    rcFile: path.join(os.homedir(), '.config', 'fish', 'completions', 'asyncapi.fish'),
+    detectFile: path.join(os.homedir(), '.config', 'fish', 'config.fish'),
```

Comment:
Detection using $SHELL or some other way would be better. Not everyone has their config like this.

Reply:
After running the npm install 
Here is the result 
<img width="687" alt="image" src="https://github.com/user-attachments/assets/52b1fc8b-7c15-49ec-b2d0-a2e3150a6f8e" />
The process.env.SHELL returning /bin/sh the because many system use by default to run these kind of scripts. 

but if we use the same concept as in the PR, we can easily detect the default shell of the os. Result
<img width="672" alt="image" src="https://github.com/user-attachments/assets/650f5d40-078e-4854-94de-0e0284352153" />